mobile/app/Components/AREAsList.tsx

```diff
@@ -0,0 +1,252 @@
+/**
+ * @fileoverview This file shows the list of areas
+ */
+
+import React, {useState, useEffect} from 'react';
+import {
+    View,
+    Text,
+    TouchableOpacity,
+    Image,
+    ScrollView,
+    ActivityIndicator,
+    Switch,
+    Animated,
+    RefreshControl
+} from 'react-native';
+import {useNavigation} from 'expo-router';
+import {styled} from 'nativewind';
+import colors from "../../constants/Colors";
+import actionReactionJsonDataService from '../Utils/actionReactionJsonData.serivce';
+import {AreaDetailsInterface} from "../Interfaces/areaData.interface";
+import {Ionicons} from "@expo/vector-icons";
+import BravmmModal from "./BravmmModal";
+import {eraseArea, toggleArea} from "../Utils/callApi";
+import {FadeLoading} from 'react-native-fade-loading';
+
+const StyledView = styled(View);
+const StyledText = styled(Text);
+const StyledTouchableOpacity = styled(TouchableOpacity);
+const StyledImage = styled(Image);
+const StyledScrollView = styled(ScrollView);
+
+const servicePicture: { [key: string]: string } = {
```

Comment:
should return ImageSourcePropType

(don't forget to import it on to of the file)